source/SIL.AppBuilder.Portal.Frontend/src/ui/styles/_base/variables.scss

```diff
@@ -0,0 +1,20 @@
+$sizeUnit: rem;
```

Comment:
semantic ui has no style utilities... so I copied some of my own from another project.

I think in the future I want to look at https://tailwindcss.com/